multitenancy/tenant-manager-datamodel/src/main/resources/META-INF/openapi.json

```diff
@@ -259,22 +262,67 @@
                     "organizationId": {
                         "description": ""
                     },
-                    "authServerUrl": {
-                        "description": "Http endpoint for the auth server (including realm) to be used for this tenant to authenticate against the registry",
-                        "type": "string"
-                    },
-                    "authClientId": {
-                        "description": "ClientId in the authentication server to be used by the registry to authenticate incoming requests made by the tenant",
-                        "type": "string"
```

Comment:
Are we thinking too specifically on this?  If others want to leverage the MT functionality, will this be flexible enough?  Not something I'm proposing we change in the near term, but something to consider for the future.

Reply:
Actually the discussion about this should happen here https://github.com/Apicurio/apicurio-registry/pull/1495

Reply:
Agree, that discussion should happen there.